extensions/WSLExtension/DevHomeProviders/WslProvider.cs

```diff
@@ -33,13 +38,20 @@ public WslProvider(IStringResource stringResource, IWslManager wslManager)
         _stringResource = stringResource;
         _wslManager = wslManager;
         Icon = new(ExtensionIcon);
+        _wslEnablementError = GetErrorStringForWhenFeatureNotEnabled();
     }
 
     /// <summary>
     /// Creates and returns the adaptive card session that will appear in the create environment creation UX in Dev Home.
     /// </summary>
     public ComputeSystemAdaptiveCardResult CreateAdaptiveCardSessionForDeveloperId(IDeveloperId developerId, ComputeSystemAdaptiveCardKind sessionKind)
     {
+        if (!string.IsNullOrEmpty(_wslEnablementError))
+        {
+            _log.Error($"Virtual machine platform optional component not enabled. A reboot is needed.");

```

Comment:
This might be getting too specific.  Is it worth separating out the logs into these two states.
1. Virtual Machine Platform optional component is not enabled
2. Virtual Machine component is enabled but the computer need to restart?